test/e2e/common.go

```diff
@@ -150,6 +150,11 @@ func DumpSpecResourcesAndCleanup(ctx context.Context, specName string, clusterPr
 			Namespace: namespace,
 		}, intervalsGetter(specName, "wait-delete-cluster")...)
 
+		// Listing bmh, m3m and machines once deletion is done
```

Comment:
Hmm. Why would we want to list them here? The checking leftover below will fail if there's still leftover, that should be enough.

Reply:
To check the state of bmh, m3m and machines if deletion fails.
Just want to see in what phase these resources would be in when the test fails.

Reply:
removed the listings.